src/ast/mod.rs

```diff
@@ -526,6 +527,14 @@ pub enum Statement {
         /// `RESTRICT` or no drop behavior at all was specified.
         cascade: bool,
     },
+    Replace {
+        /// TABLE
```

Comment:
It would be better to have this encoded in a ReplaceAction which is part of the OrAction.
Replace itself is no statement!

Reply:
I guess this implements this bit: _For compatibility with MySQL, the parser allows the use of the single keyword REPLACE as an alias for "INSERT OR REPLACE"._ I'm not sure it's appropriate to introduce a duplicate of Statement::Insert just to implement another syntax. Perhaps normalize it to `INSERT OR REPLACE` on parsing? Change `Statement::Insert::or` to a tristate-enum: None, Replace, Or(SqliteConflictResolutionAlgorithm)?

Reply:
I did try some of these methods.Finally, I think a `replace` statement is more reasonable. Because the implementation is much easier, on the other hand, one statement to support one syntax is straightforward. I don't think we should care whether it is an alias for `insert` during parsing the sql. And saying about normalize `replace` to `insert or replace`, we will lost the original details, I'm not sure is it necessary to hold the detail. If we do so, we can't build a `replace` sql from ast any longer.

Reply:
Our AST is lossy in many aspects, so unless someone _needs_ to be able to distinguish REPLACE from INSERT OR REPLACE, I'd rather we didn't add a separate variant. But I can live with either solution.